packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -300,56 +317,57 @@ class GraphiQLInternals extends React.Component<
       //   onDoubleClick={this.handleResetResize}
       //   onMouseDown={this.handleResizeStart}>
       //   <div className="queryWrap" style={queryWrapStyle}>
-      <QueryEditor
-        onHintInformationRender={this.handleHintInformationRender}
-        onClickReference={this.handleClickReference}
-        editorTheme={this.props.editorTheme}
-        readOnly={this.props.readOnly}
-        editorOptions={this.props.operationEditorOptions}
-      />
+      <section>
```

Comment:
these components already contain section tags, so this leads to `<section><section></section></section>` which is not semantic html as i understand